
<launch>
 <node pkg="pcl_tutorial" type="voxel_filter_node" name="voxel_filter_node" output="screen" >
<remap from="point_cloud_in" to="/camera/depth_registered/points" />
</node>


 <node pkg="pcl_tutorial" type="passthrough_filter_node" name="passthrough_filter_node" output="screen" >
<remap from="point_cloud_in" to="/vf_out" />
</node>

 <node pkg="pcl_tutorial" type="plane_segmentation_node" name="plane_segmentation_node" output="screen" >
<remap from="point_cloud_in" to="/pf_out2" />
</node>

 <node pkg="pcl_tutorial" type="trianglemesh_node" name="trianglemesh_node" output="screen" >
<remap from="pf_out" to="/pcl_meshtriangle" />
</node>

</launch>

